.github/workflows/docs-verify.yml

```diff
@@ -20,3 +20,9 @@ jobs:
       - name: Build docs
         run: |
           tox -edocs
+      - name: Upload docs artifact
+        if: always()
```

Comment:
This forces the build to upload even if it has errors. This is useful for debugging cryptic sphinx failures